keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -324,8 +326,13 @@ kind: Preferences
 command:
   aliases:
     - alias: getdbprod
-      command: get pods -l what=database --namespace us-2-production
-      
+      command: get pods
```

Comment:
nit: Would it make sense to also change the L:343 to `interactive` to not mislead people?

Reply:
Oh yeah, this was written before we had settled on the interactive flag for delete I believe so updating that to match would be good.

Reply:
Oh, I missed it - fixed in latest push.